mlir/tools/mlir-tblgen/OpDocGen.cpp

```diff
@@ -54,12 +54,12 @@ cl::opt<bool> allowHugoSpecificFeatures(
     cl::cat(docCat));
 
 void mlir::tblgen::emitSummary(StringRef summary, raw_ostream &os) {
-  if (!summary.empty()) {
-    StringRef trimmed = summary.trim();
-    char first = std::toupper(trimmed.front());
-    StringRef rest = trimmed.drop_front();
-    os << "\n_" << first << rest << "_\n\n";
-  }
+  if (summary.empty())
+    return;
+  StringRef trimmed = summary.trim();
+  char first = std::toupper(trimmed.front());
+  StringRef rest = trimmed.drop_front();
+  os << "\n_" << first << rest << "_\n";
```

Comment:
Is there anything that changed here or is this NFC (just using early-exit)

Reply:
Oh I spotted the removal of the extra \n!

Reply:
Ah, that was made more difficult to review thanks to format change, apologies!

Switched to early exit since some doc methods wrapped the entire body in an `if` while others used early exit, decided to make consistent.